blueprints/port-fixed-ip.yaml

```diff
@@ -2,33 +2,42 @@ tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
   - http://www.getcloudify.org/spec/cloudify/4.3.1/types.yaml
-  - ../plugin.yaml
+  - plugin:cloudify-openstack-plugin
```

Comment:
Do not you think we should stick with `../plugin.yaml` because this represent the current changes we have, we could make some changes exists on `../plugin.yaml` but still does not exists on `plugin:cloudify-openstack-plugin`

Reply:
`../plugin.yaml` is only if you are in local mode. These are manager tests. The test uploads a new wagon to the manager with current changes. So it should use that plugin.